app/models/federated_protocols/saml.rb

```diff
@@ -2,6 +2,9 @@
 
 module FederatedProtocols
   class Saml
+    IAL_PREFIX = %r{^http://idmanagement.gov/ns/assurance/ial}
```

Comment:
[suggestion] it looks like the LOA prefix is not being included. we should probably add that back as this is a refactor, not a code change

Reply:
I made this change on the basis of https://gsa-tts.slack.com/archives/C0NGESUN5/p1724965595589029.

Reply:
As far as I can tell (and the tests), it only affects the logged analytics event, and doesn't affect what we send to partners.

Reply:
I'll open a separate issue.

Reply:
thanks! once you add the LOA prefix back in i'll approve